README.uml

```diff
@@ -7,18 +7,20 @@
 # 1: when a class creates a list of other classes,
 # 2: when class names are passed by argument.
 
-# for the dynamo 0.1 api:
-pyreverse -A -p dynamo0p1 dynamo0p1.py # -A includes base classes
-dotty classes_dynamo0p1.dot # to view
-dot -Tsvg classes_dynamo0p1.dot > classes_dynamo0p1.svg # to generate svg
+# for the dynamo 0.3 api:
+pyreverse -A -p dynamo0p3 dynamo0p3.py # -A includes base classes
+dotty classes_dynamo0p3.dot # to view
+dot -Tsvg classes_dynamo0p3.dot > classes_dynamo0p3.svg # to generate svg
+dot -Tpng classes_dynamo0p3.dot > classes_dynamo0p3.png # to generate png
 
 # for gocean
 pyreverse -A -p gocean gocean.py
 dotty classes_gocean.dot
 dot -Tsvg classes_gocean.dot > classes_gocean.svg
 
-# svg can be imported into libreoffice, although there appears to be a bug with visualisation
-# (it looks OK in eog)
-
-# below might make it prettier?
-http://www.hokstad.com/making-graphviz-output-pretty-with-xsl
+# to make the topclasses.png file I manually edited the dot file to
+remove any unwanted nodes and edges then added ... style=filled,
+fillcolor="antiquewhite" ... to the nodes that I wanted to shaded. I
+also manually removed internal variables, but I'm sure there is an
+option to automate this. The resultant dot file -
+dynamo0p3_topclasses.dot - is kept in the doc directory for reference.
```

Comment:
Missing blank line at the end of the file.

Reply:
Added.